guides/walkthrough/page_1.md

```diff
@@ -0,0 +1,19 @@
+**This is the complete walkthrough of the commands of the HealthScreening Bot.**
+This bot is fairly simple and self-explanatory, but this guide serves as an complete walkthrough of the commands.
+`Highlighted text` is a command name
+- dashed text are command arguments!
+
+
+**Automated Health Screening Commands:**
+`set_auto` : Sets the information used for the automated health screening
+Paremeters:
```

Comment:
```suggestion
__Parameters__:
```

Reply:
There was a typo, so I took the liberty of fixing it.

Reply:
Thank you